src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -449,11 +449,12 @@ private function validateAndDenormalize(array $types, string $currentClass, stri
     {
         $expectedTypes = [];
         foreach ($types as $type) {
-            if (null === $data && $type->isNullable()) {
+            if (null === $data && $isNullable = $type->isNullable()) {
```

Comment:
If it is not very likely the function ends with the first check they should be switched to get rid of the isset check later. Same for `$builtinType`.